### PR TITLE
[containerized-data-importer] fix customCertificate

### PR DESCRIPTION
**Pull request by kvaps**
## Description

Make CDI working with customCertificate

## Why do we need it, and what problem does it solve?

If cluster configured to use customCertificate method, cdi module can't be installed

## What is the expected result?

CDI able to work with customCertificate

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: containerized-data-importer
type: fix
summary: Make CDI working with `customCertificate`.
```

### CODE REVIEW

File: modules/490-virtualization/images/artifact/patches/007-tolerations-for-strategy-dumper-job.patch [hunk 1]
```diff
@@ -0,0 +1,24 @@
+diff --git a/pkg/virt-operator/kubevirt.go b/pkg/virt-operator/kubevirt.go
```
Comment (apolovov) [inline modules/490-virtualization/images/artifact/patches/007-tolerations-for-strategy-dumper-job.patch:1]: It is from other pr.

Comment (kvaps) [reply modules/490-virtualization/images/artifact/patches/007-tolerations-for-strategy-dumper-job.patch:1]: fixed, thanks

**Review by duckhawk — APPROVED** [review]

File: modules/491-containerized-data-importer/openapi/doc-ru-config-values.yaml [hunk 2]
```diff
@@ -5,3 +5,49 @@ properties:
...
+          secretName:
+            description: |
+              Имя Secret'а в пространстве имён `d8-system`, который будет использоваться для web-интерфейса документации.
+
+              Secret должен быть в формате [kubernetes.io/tls](https://kubernetes.github.io/ingress-nginx/user-guide/tls/#tls-secrets).
+  https:
+    description: |
+      Тип используемого сертификата.
+
+      При использовании этого параметра полностью переопределяются глобальные настройки `global.modules.https`.
```
Comment (diafour) [inline modules/491-containerized-data-importer/openapi/doc-ru-config-values.yaml:20]:
```suggestion
      Этот параметр задаёт особые настройки https для модуля, переопределяя глобальные настройки `global.modules.https`.
```

File: modules/491-containerized-data-importer/openapi/doc-ru-config-values.yaml [hunk 3]
```diff
@@ -5,3 +5,49 @@ properties:
       Ручное управление режимом отказоустойчивости.
 
       По умолчанию режим отказоустойчивости определяется автоматически. [Подробнее](../../deckhouse-configure-global.html#параметры) про режим отказоустойчивости.
+      customCertificate:
+        description: Настройки для использования пользовательского сертификата.
+        properties:
+          secretName:
+            description: |
+              Имя Secret'а в пространстве имён `d8-system`, который будет использоваться для web-интерфейса документации.
```
Comment (diafour) [inline modules/491-containerized-data-importer/openapi/doc-ru-config-values.yaml:13]: > будет использоваться для web-интерфейса документации.

Не очень понятно про документацию, в оригинале речь про Hubble:

> to use with the Hubble web UI.

Comment (duckhawk) [reply modules/491-containerized-data-importer/openapi/doc-ru-config-values.yaml]: Вроде бы пофиксил

File: modules/491-containerized-data-importer/openapi/doc-ru-config-values.yaml (hunk 2 above)
Comment (duckhawk) [reply modules/491-containerized-data-importer/openapi/doc-ru-config-values.yaml]: Поправил

**Review by duckhawk — APPROVED** [review]

**Review by diafour — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @diafour. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [4090](https://github.com/deckhouse/deckhouse/pull/4090) to the branch [release-1.44](https://github.com/deckhouse/deckhouse/tree/release-1.44) successful!

**Pull request merged**